packages/workbench/src/app/app.routes.ts

```diff
@@ -1,3 +1,9 @@
-import { Routes } from '@angular/router';
+import {Routes} from '@angular/router';
+import {SparqlComponent} from './sparql/sparql.component';
+import {GraphqlComponent} from './graphql/graphql.component';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+  {path: 'sparql', component: SparqlComponent},
+  {path: 'graphql', component: GraphqlComponent},
+  // {path: "**", component: EmptyRouteComponent}
```

Comment:
Uncomment this line and implement the component. It should be a component with empty HTML. Without this default path definition, the Angular router will throw an exception and break navigation to the view if we click on a link leading to a view managed by the legacy workbench.